.github/workflows/deploy.yml

```diff
@@ -170,7 +170,7 @@ jobs:
         uses: ./.github/workflows/composite/npm
         with:
           registry: 'https://npm.pkg.github.com'
-          scope: '@NeuraLegion'
+          scope: '@BrightSec'
```

Comment:
We didn't change the org name in GitHub yet

Reply:
It's out of the scope for current task.
But in general: why not?
It's available: https://www.npmjs.com/package/@brightsec/cli.
And our official docs reference that: https://docs.brightsec.com/docs/installation-options#npm

Reply:
First of all, please check the registry carefully you refer to. It has nothing to npm, and all the references mentioned above are not relevant to the current situation. You have modified the scope for GPR without fully understanding the implications.

Reply:
Oh yeah, my bad. I didn't pay enough attention to github part of this host name: `npm.pkg.**github.com**`